test/e2e/storage/csi_volumes.go

```diff
@@ -153,17 +149,18 @@ var _ = utils.SIGDescribe("CSI Volumes", func() {
 			cs     clientset.Interface
 			csics  csiclient.Interface
 			driver testsuites.TestDriver
+			config *testsuites.TestConfig
 		)
 
 		BeforeEach(func() {
 			cs = f.ClientSet
 			csics = f.CSIClientSet
-			config := testsuites.TestConfig{
+			config = &testsuites.TestConfig{
```

Comment:
`perTestConfig`? ditto everywhere else. I think it needs to be really clear what the lifecycle of these objects are, thats what bit us in the butt earlier. The more self-documenting the better. But we should be even clearer still by adding a comment to each of these variables mentioning when they are created/destroyed and when/if they should be modified/cleared.

Reply:
Okay.